golos.publication/parameters.hpp

```diff
@@ -0,0 +1,69 @@
+#pragma once
+#include <eosiolib/singleton.hpp>
+#include <common/parameter.hpp>
+#include <common/parameter_ops.hpp>
+
+namespace golos {
+
+    using namespace eosio;
+
+    struct st_max_vote_changes : parameter {
+        uint8_t max_vote_changes;
+
+        void validate() const override {
+            eosio_assert(max_vote_changes > 0, "Max vote changes can't be negative.");
+        }
+    };
+    using max_vote_changes_prm = param_wrapper<st_max_vote_changes, 1>;
+
+    struct st_cashout_window : parameter {
+        uint32_t cashout_window;
+
+        void validate() const override {
+            eosio_assert(cashout_window > 0, "Cashout window can't be negative.");
```

Comment:
message is wrong, negative impossible here, this assert only denies `0`. change to `Cashout window must be greater than 0`